src/ui/page/internalError/internalError.scss

```diff
@@ -0,0 +1,131 @@
+@import '@/ui/style/fonts.scss';
+@import '@/ui/style/theme.scss';
+@import '@/ui/style/mixins.scss';
+
+.okp4-dataverse-portal-internal-error-page-main {
+  @include with-theme() {
+    height: 100%;
+
+    &.dark {
+      background: center / cover url('@/ui/asset/images/internal-error-dark.webp');
+
+      @include use-breakpoints(('mobile')) {
+        background: unset;
+      }
+
+      @include use-breakpoints(('tablet')) {
+        background-size: contain;
+        background-repeat: no-repeat;
+      }
+
+      .okp4-dataverse-portal-internal-error-page-text-wrapper {
+        @include use-breakpoints(('mobile')) {
+          background: center url('@/ui/asset/images/internal-error-dark-mobile.webp') no-repeat;
+        }
+      }
+    }
+
+    &.light {
+      background: center / cover url('@/ui/asset/images/internal-error-light.webp');
+
+      @include use-breakpoints(('mobile')) {
+        background: unset;
+      }
+
+      @include use-breakpoints(('tablet')) {
+        background-size: contain;
+        background-repeat: no-repeat;
+      }
+
+      .okp4-dataverse-portal-internal-error-page-text-wrapper {
+        @include use-breakpoints(('mobile')) {
+          background: center url('@/ui/asset/images/internal-error-light-mobile.webp') no-repeat;
+        }
+      }
+    }
+
+    .okp4-dataverse-portal-internal-error-page-content-wrapper {
+      @include columns-with-gap(28px);
+      align-items: center;
+      justify-content: center;
+      text-align: center;
+      height: 100%;
+
+      @include use-breakpoints(('mobile', 'tablet')) {
+        gap: 24px;
+      }
+
+      .okp4-dataverse-portal-internal-error-page-text-wrapper {
+        display: flex;
+        flex-direction: column;
+        gap: 28px;
+        align-items: center;
+
+        @include use-breakpoints(('mobile', 'tablet')) {
+          gap: 24px;
+        }
+
+        h1 {
+          @include norse-font(700);
+          line-height: 88px;
+          font-size: 80px;
+          color: themed('text');
+
+          @include use-breakpoints(('mobile', 'tablet')) {
+            font-size: 40px;
+            line-height: 48px;
+          }
+        }
+
+        .okp4-dataverse-portal-internal-error-page-message-titles {
+          display: flex;
+          flex-direction: column;
+          align-items: center;
+          max-width: 960px;
+
+          @include use-breakpoints(('desktop')) {
+            max-width: 750px;
+          }
+
+          .okp4-dataverse-portal-internal-error-page-message-apology {
+            opacity: 0.4;
+          }
+        }
+
+        .okp4-dataverse-portal-internal-error-page-message-suggestion {
+          @include noto-sans(300);
+          font-size: 16px;
+          color: themed('text');
+          line-height: 22px;
+
+          @include use-breakpoints(('desktop')) {
+            max-width: 554px;
+          }
+
+          @include use-breakpoints(('large-desktop')) {
```

Comment:
From what I see, you use breakpoints sometimes for mobile, assuming that the values under your class are for desktop and large desktop; except that sometimes you use breakpoints for desktop and large desktop, which means that your base code in your class is for mobile and tablet.
To keep some consistency, I suggest you start from the fact that the code you write is about the desktop, and if needed you use breakpoints for the mobile/tablet/large desktop.
Otherwise it's difficult to read the code :)

Reply:
@lolottetheclash should desktop first strategy be applied everywhere? Or by consistency you mean across a single CSS file? Mobile first is generally a recommended approach, what is the general rule about media queries we must follow on the project? Thanks

Reply:
@0xLodz I totally agree with you, we have to start on this principle from now on: Mobile first.
I will create a technical ticket to refactor the existing code in this direction.
FYI @ErikssonJoakim @AnjaMis @fredericvilcot @ccamel

Reply:
Noted! Should I then refactor the whole file to mobile-first approach?